doc-content/enterprise-only/getting-started/business-rules-conditions-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='guided-suspension-rule-conditions-proc']
+= Setting the Suspension rule conditions
+
+Set the `Suspension` rule conditions that are used to determine the driver's violation.
+
+.Prerequisite
+You have created the Driver License Suspension rule.
+
+.Procedure
+. Click (image:getting-started/6176.png[]) next to the *WHEN* label to open the *Add a condition to the rule* window. . . Select *Driver* and click *Ok*.
++
+
+.Create new Guided Rule window
+image::getting-started/add-condition.png[]
+. Click the *There is a Driver* label to open the *Modify constraints for Driver* window.
+. Enter `driver` in the *Variable name* field and click *Set*.
+. Click the *There is a Driver[driver]* and click *Expression editor*.
+. Click *[not bound]* to open the *Expression editor*.
+. In the *Bind the Expression to a new variable* field, enter: `previousPts` and click *Set*.
+. From the *Choose* menu, select *totalPoints*.
+. Click (image:getting-started/6176.png[]) next to line 1 (the *previousPts* label) to open the *Add a condition to the rule* window.
+. Select *From Accumulate* and click *Ok*.
+. Click *click to add pattern* above the *From Accumulate* label and select *Number* from the *choose fact type* pull-down menu.
+. Click *There is a number* label to open the *Modify contraints for Driver* window.
+. Click *Expression editor* and select *intValue()* from the
+*[not bound]:* *Choose* menu.
+. Click *[not bound]* to open the *Expression editor*.
+. In the *Bind the Expression to a new variable* field, enter: `totalNewPoints` and click *Set*.
+. Click *click to add pattern* and select *Violation* from the *choose fact type* pull-down menu.
+. Click *All Violation with:* to open the *Modify contraints for Violation* window and select *points* from the *Add a restriction on a field* pull-down menu.
+. Click *please choose* next to the *points* label and select *greater than*.
+. Click (image:getting-started/6191.png[]), then click *Literal value*.
+. Click the *points* label to open the *Add a field* window and enter `vPoints` and click *Set*.
```

Comment:
I'd make this a series and eliminate one and

Reply:
Done